src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -417,6 +417,8 @@ public static void MapContainerView(IViewHandler handler, IView view)
 				handler.HasContainer = viewHandler.NeedsContainer;
 			else
 				handler.HasContainer = view.NeedsContainer();
+
+			handler.UpdateValue(nameof(IView.Visibility));
```

Comment:
I think we also need to add

handler.UpdateValue(nameof(IView.Opacity))

and remove this line from LabelHandler.windows.cs

        public static void MapOpacity(ILabelHandler handler, ILabel label)
        {
            handler.UpdateValue(nameof(IViewHandler.ContainerView)); // this line
            handler.PlatformView.UpdateOpacity(label);
            handler.ToPlatform().UpdateOpacity(label);
        }
I think when "handler.UpdateValue(nameof(IViewHandler.ContainerView));" was originally added we didn't have the correct MapContainerview flow, so it made sense there

You only need to update the ContainerView if that property would result in the container getting added/removed